qiskit/providers/provider.py

```diff
@@ -28,12 +29,25 @@ class Provider:
 
     version = 0
 
+    @deprecate_func(
+        since=1.1,
+        additional_msg="The abstract Provider and ProviderV1 classes are deprecated and will"
+        " be removed in 2.0. You can just remove it as the parent class and a `get_backend` method that returns the backends from `self.backend`.",
```

Comment:
technically, there is no need to remove `get_backend`.

Reply:
Oh, I meant when you remove the parent class you should add `get_backend` back manually. The one method you get for free from the abstract class is `get_backend` so if people are dropping the abstract class as their parent they'll probably want to add that single method.

Reply:
oopss! I didnt read it properly, you are right.